tests/e2e/yaml/secured-cluster-cr-template.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: platform.stackrox.io/v1alpha1
+kind: SecuredCluster
+metadata:
+  name: stackrox-secured-cluster-services
+spec:
+  clusterName: remote
+  perNode:
+    collector:
+      collection: __COLLECTION_METHOD__
```

Comment:
I don't think this should be introduced in such form. Please follow envsubst approach, similar to central CR.

Reply:
Done